docs/tutorial/add-subscription.md

```diff
@@ -0,0 +1,106 @@
+---
+id: add-subscription
+title: Add subscription
+sidebar_label: 9. Add subscription (advanced)
+---
+
+We discovered how to Query and Mutate data, now, we are going to discover a most advanced part of GraphQL, the [event-based subscriptions](https://graphql.org/blog/subscriptions-in-graphql-and-relay/#event-based-subscriptions), well-known as `subscription` in the specification.
+
+In this tutorial, we'll implement a subscription which will wait for the end of the cooking process for a given recipe.
+
+During the cooking process, we'll notify the user each second about the remaining time, and in the end, we'll notify that the cooking is over.
+
+This feature will allow the client of our GraphQL API to update its UI in real time.
+
+For development purposes, we'll consider that the cooking duration is in seconds instead of minutes _(Our over is really powerful !)_.
```

Comment:
```suggestion
For development purposes, we'll consider that the cooking duration is in seconds instead of minutes _(Our oven is really powerful !)_.
```